prep-webapp.sh

```
@@ -0,0 +1,57 @@
+
+echo "Building packages..."
+nix develop -i ${keep[@]} -c bash -c '(npm run build -w @rainlanguage/orderbook)'
+nix develop -i ${keep[@]} -c bash -c '(npm run build -w @rainlanguage/ui-components && npm run build -w @rainlanguage/webapp)'
+
+# Temporarily disable command echoing
+set +x
+
+export LANG=en_US.UTF-8
+export LC_ALL=en_US.UTF-8
+
+GREEN='\033[0;32m'
```

Shellcheck (0.11.0)

[warning] GREEN appears unused. Verify use (or export if used externally).

(SC2034)